package.json

```diff
@@ -54,4 +54,4 @@
     "test:apex": "sh ./scripts/runLocalTests.sh",
     "test:lwc": "sfdx-lwc-jest --coverage --skipApiVersionCheck"
   }
-}
\ No newline at end of file
+}
```

Comment:
this should be reverted

Reply:
The entire file?

Reply:
This comment is at the line level, and is specifically in regards to the CR/LF change that was made to the end of the file